docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -103,8 +108,8 @@ test('Button click', async t => {
 
     await t
         .click(btn)
-        // Regular assertion will fail immediately, but TestCafe retries to run DOM state
-        // assertions many times until this assertion pass successfully within the timeout.
+        // A regular assertion will fail immediately, but TestCafe retries to run DOM state
+        // assertions many times within the timeout until this assertion passes successfully.
         // The default timeout is 3000 ms.
         .expect(btn.textContent).contains('Loading...');
```

Comment:
We should add info about query mechanism for client function in https://devexpress.github.io/testcafe/documentation/test-api/assertions/#smart-assertion-query-mechanism section
E.g. in sentence `If the TestCafe assertion receives a Selector's DOM node state property as an actual value` and may be in other places.
And https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#--assertion-timeout-ms
and for `assertionTimeout` here https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html#run